pilot/pkg/xds/ads.go

```diff
@@ -508,16 +504,15 @@ func (s *DiscoveryServer) initConnection(node *core.Node, con *Connection) error
 		con.proxy.VerifiedIdentity = id
 	}
 
-	// Register the connection; this allows pushes to be triggered for the proxy. Note: the timing of
-	// this an initProxyState is important. While registering for pushes *after* initialization is
```

Comment:
seems like the "and initProxyState" is important part of the comment, despite a typo